aws/client.go

```diff
@@ -81,6 +82,9 @@ func (t *ResilientTransport) tries(req *http.Request) (res *http.Response, err e
 		if !t.ShouldRetry(req, res, err) {
 			break
 		}
+		if try == (t.MaxTries-1) {
+			break
+		}
```

Comment:
related to https://github.com/mitchellh/packer/issues/2001 - No message due to the bug fixed by theese lines